services/data-collection/native-cookie-support.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Native Cookie Support
+description: Manage analytics cookies directly at the edge for unified user tracking.
+icon: cookie
+---
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+Native Cookie Support is a powerful feature that allows Edgee to manage analytics cookies directly at the edge, 
```

Comment:
(avoid using "allows" twice in the same sentence)

"Native Cookie Support is a powerful feature to manage analytics cookies directly at the edge"